designer/client/list-items.js

```diff
@@ -67,7 +67,7 @@ const SortableList = SortableContainer(({ items, conditions, type, removeItem, o
   return (
     <tbody className='govuk-table__body'>
       {items.map((item, index) => (
-        <SortableItem key={`item-${index}`} item={item} index={index} conditions={conditions} type={type} onBlur={onBlur} removeItem={removeItem} />
+        <SortableItem key={`item-${item.value}`} item={item} index={index} conditions={conditions} type={type} onBlur={onBlur} removeItem={removeItem} />
```

Comment:
Hmm.. seems like `index` would be a better variable to key on as it's guaranteed to be unique. What happens if I have two list items with the same value?

Reply:
I don't think this is the reason why sorting of list items doesn't work. Have you tried it in the site with this change to check that it fixes the issue? 
I documented the reason for this change and the issue I was seeing here: https://github.com/XGovFormBuilder/digital-form-builder/issues/25